### DISCUSSION TITLE
Enable Nether Roof Building for Bedrock User

### DISCUSSION THREAD

**Discussion by EshXKun** [Stardust-Labs-MC/StardustMC#7] [Ideas]
I would really like this feature be enabled since it's making hard for bedrock players to have gold in huge amount and other essential farms.
I would also like to request the command "/geyser offhand"  which enable bedrock players to hold item in offhand thus making it easier for us to repair tools

**Comment by aisukuma** [comment]
So the setting for Bedrock users to build above the nether is already enabled, I'm not sure what seems to be the issue. Though geyser is on the proxy and not on the server, but will have to further investigate this in [this issue](https://github.com/Stardust-Labs-MC/StardustMC/issues/9).
